testing/unit-testing-jest/docs/getting-dom-elements.md

```diff
@@ -0,0 +1,63 @@
+
+# How to get elements from the DOM on testing
+Check if an element render the correct information on the page is very important to test and more if we are frontend developers, so to test the elements you need to access to the elements and also call his events.
+
+## Getting element
+Consider the next element
+test.html
```

Comment:
Remember that if you do this, github will display it like:
Consider the next element test.html

Reply:
Check this [docs](https://docs.github.com/es/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)